src/main/java/minijava/ir/optimize/AliasAnalyzer.java

```diff
@@ -824,6 +837,7 @@ public boolean transform() {
       Node ptr = sideEffect.getPred(1);
       Set<IndirectAccess> aliasClass = getPointsTo(ptr);
       boolean isLoad = sideEffect.getOpCode() == iro_Load;
+      assert !aliasClass.isEmpty() : "Empty alias class at " + sideEffect;
```

Comment:
Judging from where I had to fight bottlenecks so far, assertions never were a concern.

I think we should disable them nonetheless for the benchmarks, because it would be really annoying if our compiler broke because of one of them. For example in this case it might be possible to continue, taking the odds of incorrectly generated code (which only really broke for the `big/math.java` test case).